Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript

```diff
@@ -87,6 +87,7 @@ on updateColorValue pColor
    lock screen
    lock messages
    set the swatchColor of widget "color" of me to pColor
+   put pColor into field "value" of me
```

Comment:
@livecodeali 
I updated the pull request and propose now to take handler "updateColorValue" out and change in 
handler "updateSubeditorValue" to "updateColorDisplay"
As far as I can tell it works and has no untoward effects